packages/blockchain-info-components/src/Form/DateInput/index.js

```diff
@@ -168,7 +169,7 @@ const BaseDateInput = styled(ReactDatetime)`
   .rdtPicker th.rdtDisabled,
   .rdtPicker th.rdtDisabled:hover {
     background: none;
-    color: ${props => props.theme['gray-2']};
+    color: ${props => props.theme['gray-1']};
     cursor: not-allowed;
```

Comment:
Do we mind all the other hex codes in here?

Reply:
there is a few left yes, but are not very relevant I think (border-shadows etc...)

Reply:
I mean I tried switching the theme and I did not see anything shocking :)